.flowconfig

```diff
@@ -0,0 +1,34 @@
+[ignore]
```

Comment:
That means you can't run flow at the top level which kind of sucks. It also means you have to `../` up to things like the custom flow definition files and make special cases for node_modules that you do want to bring in flow for.